### PR TITLE
Align on snake_case instead of camelCase

### LINKED ISSUE #120: Mix-matched cases and other nits
> It's silly to have both kLenInBytes and kLenInBits.  Just pick one and 
> compute the other from it.
>
> Just from an aesthetic point of view, camel case or snake case, pick one.

>  - All occurrences of "an byte string of length 32 bytes" -> "a 32-byte 
> string"

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #120

### CODE REVIEW

**Comment by jedisct1** [comment]
LGTM.

The ref implementation would not closely match the spec any more, but that can always be fixed later.

File: draft-irtf-cfrg-rsa-blind-signatures.md [hunk 1]
```diff
@@ -223,45 +223,45 @@ input_msg = Prepare(msg)
...
+public key `pk` by invoking the RSASSA-PSS-VERIFY routine defined in
 {{Section 8.1.2 of !RFC8017}}. The Finalize function performs this check before returning the signature.
 See {{verification}} for more details about verifying signatures produced through this protocol.
 
 In pictures, the protocol runs as follows:
 
 ~~~
-   Client(pkS, msg)                      Server(skS, pkS)
+   Client(pk, msg)                      Server(sk, pk)
   -------------------------------------------------------
```
Comment (jedisct1) [inline draft-irtf-cfrg-rsa-blind-signatures.md:252]: There are too many dashes now.

**Review by jedisct1 — APPROVED** [review]

File: draft-irtf-cfrg-rsa-blind-signatures.md (hunk 1 above)
Comment (chris-wood) [reply draft-irtf-cfrg-rsa-blind-signatures.md]: Fixed!

**Comment by chris-wood** [comment]
@jedisct1 yeah, agreed, we can update the reference implementation in parallel.

**Review by FredericJacobs — APPROVED** [review]

**Pull request merged**